products/sle/main.pm

```diff
@@ -572,6 +572,7 @@ sub load_consoletests() {
         loadtest "console/vim.pm";
         if (!is_staging()) {
             loadtest "console/firewall_enabled.pm";
+            loadtest "console/zypper_lifecycle.pm";
```

Comment:
Does it really have to run in that many scenarios? It seems to be pretty generic test that can easily run in console extra tests